proto/stride/stakeibc/trade_route.proto

```diff
@@ -84,7 +89,18 @@ message TradeRoute {
   // zone, back to the host zone. This is the channel ID on the trade zone side
   string trade_to_host_channel_id = 11;
 
+  // Minimum amount of reward token that must be accumulated before
+  // the tokens are transferred to the trade ICA
+  string min_transfer_amount = 13 [
```

Comment:
I'm confused why this wasn't here before? Were we using a variable which was on the trade_config like min_swap amount and so now this transfer specific functionality needs a new variable to track it?

Reply:
Yeah exactly. We just had a MinSwap and we used it both for the transfer and the actual swap

but since this deprecates the config, I just figured we’d pull it out and rename it to it’s actual use

Reply:
Yeah i dont think any need for a minimum in the other direction